src/Dapper.AOT.Analyzers/CodeAnalysis/DapperInterceptorGenerator.cs

```diff
@@ -13,6 +13,8 @@
 using System.Collections.Immutable;
 using System.Data;
 using System.Linq;
+using System.Reflection;
+using System.Runtime.InteropServices.ComTypes;
```

Comment:
double-check this maybe? is this a "devenv being devenv" thing?

Reply:
that is me being not careful, sorry! fixed